plutus-core/plutus-ir/test/transform/nameCapture/nameCapture.golden

```diff
@@ -0,0 +1 @@
+"Let nameCapture:20:1-54:1 NonRec (TermBind nameCapture:22:5-26:5 Strict (VarDecl {_varDeclAnn = nameCapture:24:9-24:33, _varDeclName = Name {_nameText = \"a\", _nameUnique = Unique {unUnique = 4}}, _varDeclType = TyBuiltin nameCapture:24:20-24:32 (SomeTypeIn DefaultUniInteger)}) (Constant nameCapture:25:9-25:24 (Some (ValueOf DefaultUniInteger 10))) :| [TermBind nameCapture:32:5-36:5 Strict (VarDecl {_varDeclAnn = nameCapture:34:9-34:33, _varDeclName = Name {_nameText = \"c\", _nameUnique = Unique {unUnique = 6}}, _varDeclType = TyBuiltin nameCapture:34:20-34:32 (SomeTypeIn DefaultUniInteger)}) (Apply nameCapture:35:9-35:34 (Apply nameCapture:35:9-35:34 (Builtin nameCapture:35:10-35:29 AddInteger) (Var nameCapture:30:9-30:9 (Name {_nameText = \"a\", _nameUnique = Unique {unUnique = 4}}))) (Var nameCapture:35:33-35:33 (Name {_nameText = \"a\", _nameUnique = Unique {unUnique = 4}})))]) (Let nameCapture:44:9-52:13 NonRec (TermBind nameCapture:46:13-50:13 Strict (VarDecl {_varDeclAnn = nameCapture:48:17-48:41, _varDeclName = Name {_nameText = \"a\", _nameUnique = Unique {unUnique = 8}}, _varDeclType = TyBuiltin nameCapture:48:28-48:40 (SomeTypeIn DefaultUniInteger)}) (Constant nameCapture:49:17-49:31 (Some (ValueOf DefaultUniInteger 7))) :| []) (Apply nameCapture:51:13-51:63 (Apply nameCapture:51:13-51:63 (Builtin nameCapture:51:14-51:33 AddInteger) (Apply nameCapture:51:35-51:60 (Apply nameCapture:51:35-51:60 (Builtin nameCapture:51:36-51:55 AddInteger) (Var nameCapture:42:13-42:13 (Name {_nameText = \"a\", _nameUnique = Unique {unUnique = 4}}))) (Var nameCapture:51:59-51:59 (Name {_nameText = \"a\", _nameUnique = Unique {unUnique = 8}})))) (Var nameCapture:51:62-51:62 (Name {_nameText = \"a\", _nameUnique = Unique {unUnique = 8}}))))"
```

Comment:
you can print the syntax properly showing uniques. can't remember how off the top of my head, but it's there

Reply:
I tried multiple variants of [prettyPlcClassicDef](https://github.com/input-output-hk/plutus/blob/6485b75dd79e3a14ad27d805b6a84c7b3cb2665b/plutus-core/plutus-core/src/PlutusCore/Pretty/Plc.hs#L118) but they don't give much improvements. Or are you talking about other ways to configure the pretty printing?

Reply:
maybe @effectfully knows the best way?

Reply:
I've updated this file to have readable syntax and removed the other one with the same name which didn't seem to be referenced from anywhere. Basically, if you need readable PIR, use `prettyPirReadable` from `PlutusIR.Core.Instance.Pretty.Readable`. Pushed.